.changeset/violet-suns-scream.md

```diff
@@ -0,0 +1,8 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Aligned private properties of internal modifiers to follow a standardized notation
```

Comment:
```suggestion
Aligned private properties of the HDS modifiers to follow a standardized notation
```
The tooltip one isn't private, we provide docs on how to use it: https://helios.hashicorp.design/components/tooltip?tab=code#ember-modifier